src/wp-includes/script-loader.php

```diff
@@ -279,6 +279,15 @@ function wp_default_packages_scripts( $scripts ) {
 	 */
 	$assets = include ABSPATH . WPINC . "/assets/script-loader-packages{$suffix}.php";
 
+	// Add the private version of the Interactivity API manually.
+	$scripts->add(
+		'wp-interactivity',
+		'/wp-includes/js/dist/interactivity.min.js',
+		array(),
+		'6.4.0',
+		1
+	);
```

Comment:
Per https://github.com/WordPress/wordpress-develop/pull/5262#pullrequestreview-1642858420, I think we should apply `defer` here, as it would be a safe change that improves performance.
```suggestion
	// Add the private version of the Interactivity API manually.
	$scripts->add(
		'wp-interactivity',
		'/wp-includes/js/dist/interactivity.min.js',
		array(),
		'6.4.0',
	);
	did_action( 'init' ) && $scripts->add_data( 'wp-interactivity', 'strategy', 'defer' );
```

cc @westonruter